CONTRIBUTING.md

```diff
@@ -11,14 +11,14 @@ To test the generator from this source code, run the following from the
 checkout of this codebase:
 
     npm uninstall -g generator-tinymce
-    npm install
-    npm run prepublishOnly
-    npm link
+    yarn install
+    yarn prepublishOnly
+    yarn link
 
 Note that you will need `yarn` installed. We may remove that requirement
 in the future. 
```

Comment:
```suggestion
Note that you will need `yarn` installed. 
```
It kind of seems odd to me to include this? Also wondering if we should move this note _before_ the commands.